pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/PulsarKafkaConnectSinkConfig.java

```diff
@@ -94,6 +94,12 @@ public class PulsarKafkaConnectSinkConfig implements Serializable {
                     + "In some cases it may result in topic name collisions (topic_a and topic.a will become the same)")
     private boolean sanitizeTopicName = false;
 
+    @FieldDoc(
+            defaultValue = "false",
+            help = "Supply kafka record with topic name without -partition- suffix for partitioned topics. \n"
+                    + "Thi si sa workaround for https://github.com/apache/pulsar/issues/19922")
```

Comment:
I am not sure we want to keep this reference here as it will go into generated docs.
We can keep the reference as a regular comment in the code